### PR TITLE
FOCUS #4: Resource Name spec changes and supporting documents

### PR DESCRIPTION
**Pull request by udam-f2**
Resource Name spec changes

### CODE REVIEW

File: supporting_content/dimensions/resourcename.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+|-----------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AWS       | CUR                      | Not present as a separate column in data. <br>Name available in some cases as a part of line_item_resource_id (in the ARN OR as the id for things like s3 buckets)                  |
+| GCP       | Big Query Billing Export | Not present as a separate column in data. <br>Column resource.global_name seems to have a resource name at the very end of the fully qualified resource name (resource.global_name). |
+| Microsoft | Cost details             | ResourceName                                                                                                                                                                     |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+See supporting content for Issue #4 (ResourceId). In some cases, the resource name is provided in the resource id as a part of the fully-qualified name
```
Comment (flanakin) [inline supporting_content/dimensions/resourcename.md:17]: Do we want to make a reference to temporary issues in GitHub?

Comment (flanakin) [inline supporting_content/dimensions/resourcename.md:17]:
```suggestion
See supporting content for issue #4 (ResourceId). In some cases, the resource name is provided in the resource ID as a part of the fully-qualified name.
```

**Review by flanakin — APPROVED** [review]

File: supporting_content/dimensions/resourcename.md (hunk 1 above)
Comment (mike-finopsorg) [reply supporting_content/dimensions/resourcename.md:17]: I think we should avoid it. Also as its currently formatted it won't link so its unclear what Issue #4 to look at.

Comment (udam-f2) [reply supporting_content/dimensions/resourcename.md:17]: Used a markdown link to reference the resourceid.md inside the supporting content (will only work after resource id is merged).

**Pull request merged**